### PR TITLE
Adds AnswerRange Elements

### PR DESCRIPTION
**Pull request by kevinkirkl3y**

### CODE REVIEW

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -11268,6 +11274,40 @@
...
+      },
+      "AnswerRangeMin": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementInteger"
+          },
+          {
+            "type": "object",
+            "description": "Minimum range of acceptable input.",
+            "x-ob-item-type": "IntegerItemType",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:11286]: I think these should be TaxonomyElementNumber, to allow for integer and float values.

Comment (jrippingale) [reply Master-OB-OpenAPI.json:11286]: We do needs to be a number to include Float values.

Comment (kevinkirkl3y) [reply Master-OB-OpenAPI.json:11286]: Are we missing a NumberItemType? I'm finding integer, decimal but no number item type.

**Comment by kevinkirkl3y** [comment]
Resolving these conflicts was becoming more cumbersome than recreating the work. Changes requested are addressed int pull 164

File: Master-OB-OpenAPI.json (hunk 1 above)
Comment (cwhanse) [reply Master-OB-OpenAPI.json:11286]: Use DecimalItemType

**Comment by cwhanse** [comment]
Superceded by #164

**Pull request closed without merging**